(DOCSP-30632): Migrate calculate fields (#19)
mongodb/docs-relational-migrator: source/mapping-rules/calculated-fields/delete-calculated-fields.txt

.. _rm-delete-calculated-fields:

========================
Delete Calculated Fields
========================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

You can delete an existing calculated field and remove the field from
your mapping rule. Deleted calculated fields do not update in future
sync job runs.

Steps
-----

> To delete a calculated field:

Reviewer: (step intro)
Please delete step intro.

#. From the :guilabel:`Mapping` screen, click a table or collection name
   on the :guilabel:`Schema Model` pane or diagram view.

#. Click the pencil icon next to the mapping rule that contains the
   calculated field you want to delete.

#. Click the vertical :guilabel:`...` icon next to the calculated field you want
   to delete.

#. Click :guilabel:`Delete calculated field`.

#. Click :guilabel:`Delete` to confirm you want to delete the field.

#. Click :guilabel:`Save and close`.

> The next time that you run a sync job, the deleted field will not be

Reviewer: ```suggestion
The next time that you run a sync job, the deleted field is not
```
